swap_meet/decor.py

```diff
@@ -1,2 +1,14 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Decor(Item):
+
+    def __init__(self, id=None, condition=0, age=0, width=0, length=0):
+        super().__init__(id, condition, age)
+        self.width = width if width ==0 else width
+        self.length = length if length ==0 else length
```

Comment:
Note that for `width` and height`, the defaults are immutable values, so we can use them directly without needing to check for a sentinel value. This would be similar to what you did with `fabric` in `Clothing`.